packages/snaps-controllers/src/snaps/SnapController.test.tsx

```diff
@@ -3898,6 +3898,455 @@ describe('SnapController', () => {
     snapController.destroy();
   });
 
+  it('throws if OnAssetsLookup handler returns an invalid response', async () => {
```

Comment:
Maybe move these and the `onAssetsConversion` tests into separate `describe` blocks for readability.